examples/main/main.cpp

```diff
@@ -392,10 +399,7 @@ int main(int argc, char ** argv) {
         sigint_action.sa_flags = 0;
         sigaction(SIGINT, &sigint_action, NULL);
 #elif defined (_WIN32)
-        auto console_ctrl_handler = +[](DWORD ctrl_type) -> BOOL {
-            return (ctrl_type == CTRL_C_EVENT) ? (sigint_handler(SIGINT), true) : false;
-        };
-        SetConsoleCtrlHandler(reinterpret_cast<PHANDLER_ROUTINE>(console_ctrl_handler), true);
+		SetConsoleCtrlHandler(reinterpret_cast<PHANDLER_ROUTINE>(ConsoleCtrlHandler), TRUE);
```

Comment:
Use spaces, not tabs. Also, what was wrong with the lambda?

Reply:
MSVC 2015 cannot handle + in lambda. More info: https://stackoverflow.com/questions/26220448/operator-on-lambda